blockchain/message_pool/src/msgpool.rs

```diff
@@ -157,17 +157,18 @@ where
 }
 
 /// This is the main MessagePool struct
+#[allow(clippy::box_vec)]
 pub struct MessagePool<T: 'static> {
     local_addrs: Vec<Address>,
-    pending: Arc<RwLock<HashMap<Address, MsgSet>>>, // mutex this
+    pending: Arc<RwLock<HashMap<Address, MsgSet>>>,
     pub cur_tipset: Arc<RwLock<Tipset>>,
     api: Arc<RwLock<T>>,
     pub min_gas_price: BigInt,
     pub max_tx_pool_size: i64,
     pub network_name: String,
-    bls_sig_cache: Arc<RwLock<LruCache<Cid, Signature>>>, // mutex this
+    bls_sig_cache: Arc<RwLock<LruCache<Cid, Signature>>>,
     sig_val_cache: LruCache<Cid, ()>,
-    local_msgs: Vec<SignedMessage>,
+    local_msgs: Box<Vec<SignedMessage>>,
```

Comment:
Why put a vec into a box? Vec is already heap allocated

Reply:
yikes, totally forgot about that, will fix